packages/editor-ui/src/views/WorkflowsView.vue

```diff
@@ -152,6 +152,9 @@ const WorkflowsView = defineComponent({
 	},
 	async mounted() {
 		this.documentTitle.set(this.$locale.baseText('workflows.heading'));
+
+		await this.tagsStore.fetchAll();
```

Comment:
Nit: If the page is loaded with empty `tags` parameter (or an un-existing value) the query part just stays in the url even tho no filters are applied, it would be nice to remove it in this case:
![image](https://github.com/user-attachments/assets/d56f1fe7-4098-4a2a-8ed8-acce666a94a8)